deps/spidershim/src/v8objecttemplate.cc

```diff
@@ -383,4 +386,22 @@ Local<FunctionTemplate> ObjectTemplate::GetConstructor() {
   return internal::Local<FunctionTemplate>::NewTemplate(isolate, ctorVal);
 }
 
+bool ObjectTemplate::IsObjectFromTemplate(Local<Object> object) {
+  return !!(JS_GetClass(GetObject(object))->flags & InstanceClass::instantiatedFromTemplate);
+}
+
+Local<FunctionTemplate> ObjectTemplate::GetObjectTemplateConstructor(Local<Object> object) {
```

Comment:
Document in the header that the arg should have tested true for IsObjectFromTemplate?